### PR TITLE
displayMyOrdersButton & displayUpdateOrderButton & displayCancelOrderButton props

### PR DESCRIPTION
**Pull request by khrizzcristian**
#### What is the purpose of this pull request?
- displayMyOrdersButton prop to op-order-options. Show or hide "My orders" button from Order placed page. (default true)
- displayUpdateOrderButton prop to op-order-options. Show or hide "Update Order" button from Order placed page. (default true)
- displayCancelOrderButton prop to op-order-options. Show or hide "Cancel Order" button from Order placed page. (default true)

#### What problem is this solving?
It allows us to hide the "update order", "cancel order" and "my orders" buttons if we don't want them on the page

#### How should this be manually tested?
Working example: https://csss--melimeloparis.myvtex.com/checkout/orderPlaced/?og=v536599meli&__bindingAddress=www.melimeloparis.ro/

Updated example with displayCancelOrderButton active: https://longo--customsoft.myvtex.com/checkout/orderPlaced/?og=1108320066749

#### Screenshots or example usage
```
"op-order-options": {
     "props": {
          "displayUpdateOrderButton": false,
          "displayMyOrdersButton": false,
          "displayCancelOrderButton": false
     }
},
```

#### Types of changes
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Requires change to documentation, which has been updated accordingly.

### CODE REVIEW

File: docs/README.md [hunk 1]
```diff
@@ -493,6 +493,8 @@ Renders the customer information. Must be placed inside an [`op-order`](#op-orde
 | Prop name   | Type      | Description                                         | Default value |
 | ----------- | --------- | --------------------------------------------------- | ------------- |
 | `fullWidth` | `boolean` | Make the options wrapper take full horizontal space | `false`       |
+| `displayMyOrdersButton` | `boolean` | Hide or show "My Orders" button | `true`       |
+| `displayUpdateOrderButton` | `boolean` | Hide or show "Update order" button | `true`       |
```
Comment (guerreirobeatriz) [inline docs/README.md:497]:
```suggestion
| `displayMyOrdersButton` | `boolean` | Whether the `My Orders` button should be displayed (`true`) or not (`false`). | `true`       |
| `displayUpdateOrderButton` | `boolean` | Whether the `Update order` button should be displayed (`true`) or not (`false`). | `true`       |
```

**Review by guerreirobeatriz — CHANGES_REQUESTED** [review]

**Review by guerreirobeatriz — APPROVED** [review]

**Comment by khrizzcristian** [comment]
New test link with displayCancelOrderButton active: https://longo--customsoft.myvtex.com/checkout/orderPlaced/?og=1108320066749

**Review by camilavcoutinho — APPROVED** [review]
For Localization, LGTM!

**Comment by klzns** [comment]
Thanks for the pull request.

Alternative approach for this issue was created:
https://github.com/vtex-apps/order-details/pull/24

**Pull request closed without merging**